### DISCUSSION TITLE
Having access to recent pastes links

### DISCUSSION THREAD

**Discussion by farhaduneci** [collove/pasteme#69] [Ideas]
Hey there,
I believe there must be a missing feature in all _pastebin_ websites that doesn't exist, and that's having access to recent paste links. This can be done via JS in the browser side or via Python in the back-end. I think using Redis or any caching method is what we need.

> Please leave your comments/ideas on this, I truly like to see something like this in PasteMe because I always loose my paste links and I'm tired of re-pasting over and over again :smile:

**Comment by lnxpy** [comment]
That's awesome. What do you think about keeping them in local storage since we have a limited account on PythonAnywhere? Is it logically right? Like keeping the last two or three snippets and popping up them when the user hits the home page (like a toast message or something).

**Comment by lnxpy** [comment]
We definitely need that idea implemented!

**Comment by farhaduneci** [comment]
#73 resolves this one.
